### PR TITLE
Set activity and use codey coin emojis

### PR DESCRIPTION
**Pull request by KevinBacabac**
Summary of Changes
Resolves these issues.
https://github.com/uwcsc/codeybot/issues/253
https://github.com/uwcsc/codeybot/issues/254

Steps to Reproduce
The bot's status is Playing CSC | .help
image

Play blackjack and confirm codey coins appear properly.
image

### CODE REVIEW

**Comment by mcpenguin** [comment]
Pls provide screenshots, thanks :))

**Comment by KevinBacabac** [comment]
Added

**Comment by mcpenguin** [comment]
when running .coin info:
image
pls fix bug thankss

**Comment by botatooo** [comment]
also the linter is angy

**Comment by KevinBacabac** [comment]
I don't know why the linter is still angry
/home/runner/work/codeybot/codeybot/src/commandDetails/coin/adjust.ts
Error:   49:82  error  Replace `user.username` with `⏎····user.username⏎··`  prettier/prettier

File: src/commandDetails/coin/info.ts [hunk 1]
```diff
@@ -6,25 +6,27 @@ import {
   SapphireMessageExecuteType,
   SapphireMessageResponse
 } from '../../codeyCommand';
+import { getCoinEmoji } from '../../components/emojis';
 import { EMBED_COLOUR } from '../../utils/embeds';
 
 // Get information about coin
+const COIN = getCoinEmoji();
 const infoEmbed = new MessageEmbed()
```
Comment (Picowchew) [inline src/commandDetails/coin/info.ts:14]: My guess as to why "undefined" appeared in the coin info embed is that these constants are initialized before `initEmojis()` in `ready.ts` is completed (I don't currently have my dev environment with me to verify). Would the custom emoji show if you try placing these 2 constants into `coinInfoExecuteCommand` instead?

**Review by Picowchew — CHANGES_REQUESTED** [review]
Thanks for this PR! :) Prettier is telling you to write the code on more lines. You can do `yarn format` for automatic formatting. I also left a comment.

File: src/commandDetails/coin/info.ts (hunk 1 above)
Comment (KevinBacabac) [reply src/commandDetails/coin/info.ts]: Thanks it worked! The linter's nice

Comment (Picowchew) [reply src/commandDetails/coin/info.ts]: Glad it worked! :)

**Review by Picowchew — APPROVED** [review]
LGTM!

**Pull request merged**